lib/metadata.go

```diff
@@ -16,6 +16,7 @@ const (
 	// container archive
 	ConfigDumpFile             = "config.dump"
 	SpecDumpFile               = "spec.dump"
+	StatusDumpFile             = "status.dump"
```

Comment:
What is the content/purpose of this file? How is it going to be used during restore?

Reply:
I updated the commit message.